### ISSUE TITLE
new script: Lab report

### ISSUE THREAD

**Issue by tylerecouture** [timberline-secondary/hackerspace-scripts-2#92]
Script scans a workstation or "all" and for each one reports:
- [x] successful connection or not (i.e. DNS is not configured)
- [x] date of last puppet run (can be seen in /etc/motd)
- [x] last boot date/time
- [x] amount of hardrive space remaining
- [ ] other useful stuff?

if possible, suppress the SSH connection output and just output those 3 lines with succinct info for each computer.

**Comment by punctuations** [comment]
for the SSH connection output I can just run all the connections and then wait till the connections have finished and print the report so at the end it will look like this:

```
------- tbl-h10-2 -------
Success: true
Last Puppet run: Tue 08 Nov 2022 06:47:06 PM PST
Last boot: Wed 09 Nov 2022 07:37:06 AM PST
Available space: 32% (46 MB) / 68% (60.72 MB)
Kernel: 5.4.0-128-generic [pulled from uname -r]
CPU Usage: 82%
Power Usage: 7.3 kW [this needs an external package added through puppet to get this info]
-------------------------

**Comment by tylerecouture** [comment]
Excellent. Don't worry about the power usage.  Why two numbers on available space?

Also be more specific what "success" means Success at what?

**Comment by punctuations** [comment]
one is avaliable and one is used (I threw in both as I thought it would be helpful)

**Comment by punctuations** [comment]
and yea I changed success to "Connection status"

**Comment by tylerecouture** [comment]
> one is avaliable and one is used (I threw in both as I thought it would be helpful)

Ok, maybe change to "32% (46 MB / 60.72 MB)"

I couldn't understand what 32% "out of" 68% meant!

**Comment by punctuations** [comment]
what about `Available space: 46 MB (68% used)`

**Comment by tylerecouture** [comment]
@punctuations After the first iteration of the Lab Report, here's what I get if I try to run a lab report on a computer that's currently mining:
![image](https://user-images.githubusercontent.com/10604391/201781296-e03f9510-4eca-48b5-9566-ae4e8b05357c.png)

**Comment by tylerecouture** [comment]
Also, the available space is wrong.  This is showing the available space on the file server (home drives) not the local hard drive.
![image](https://user-images.githubusercontent.com/10604391/201781510-5c1fb6fc-7af8-489f-ab44-2b70de08bbd3.png)

This is the local hard drive:
![image](https://user-images.githubusercontent.com/10604391/201781704-88d775fb-d381-4b22-bf19-ba0e350e7aea.png)

**Comment by tylerecouture** [comment]
Here's a command to get it specifically:
`df -H /dev/nvme0n1p2`

![image](https://user-images.githubusercontent.com/10604391/201782048-649fae64-f250-4a1a-8044-cb37348e5fa5.png)

**Comment by tylerecouture** [comment]
Everything else looks pretty awesome!

**Comment by punctuations** [comment]
is this now closed as per #95 ?
